### ISSUE TITLE
Need fix for numpy >= 1.24

### ISSUE THREAD

**Issue by nicogodet** [PANOimagen/profiletool#85]
```python
2023-04-26T14:04:35     WARNING    Traceback (most recent call last):
              File "C:\Users/godet/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\profiletool\tools\ptmaptool.py", line 87, in rightClicked
              self.profiletool.clearProfil()
              File "C:\Users/godet/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\profiletool\tools\profiletool_core.py", line 116, in clearProfil
              self.updateProfilFromFeatures(None, [])
              File "C:\Users/godet/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\profiletool\tools\profiletool_core.py", line 156, in updateProfilFromFeatures
              self.updateProfil(pointstoDraw, False, plotProfil)
              File "C:\Users/godet/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\profiletool\tools\profiletool_core.py", line 205, in updateProfil
              self.plotProfil()
              File "C:\Users/godet/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\profiletool\tools\profiletool_core.py", line 235, in plotProfil
              profile["plot_x"], profile["plot_y"] = profile_func(profile)
              File "C:\Users/godet/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\profiletool\tools\profilers.py", line 44, in slopes_pct
              x = np.array(p["l"], dtype=np.float)
              File "C:\OSGeo4W\apps\Python39\lib\site-packages\numpy\__init__.py", line 284, in __getattr__
              raise AttributeError("module {!r} has no attribute "
             AttributeError: module 'numpy' has no attribute 'float'
```

**Comment by JaviBecerra** [comment]
Hi @nicogodet,
Not sure how to reproduce this, but I think that np.float has been deprecated and you can use standard Python's `float` instead.
Cheers,
Javi

**Comment by nicogodet** [comment]
@JaviBecerra Indeed, `numpy.float` is deprecated (warning since 1.20, remove in 1.24) and builtins `float` should be used.
This issue is just a reminder, i will patch it later

**Comment by nicogodet** [comment]
Fixed in #88
